tests_pycsob/test_api.py

```diff
@@ -1,54 +1,57 @@
 # coding: utf-8
-import os
 import json
+import os
 import pytest
+import responses
 from collections import OrderedDict
-from requests.exceptions import HTTPError
+from datetime import datetime
+from freezegun import freeze_time
 from unittest import TestCase
-from urllib3_mock import Responses
+from requests.exceptions import HTTPError
 
 from pycsob import conf, utils
 from pycsob.client import CsobClient
 
+
+BASE_URL = 'https://gw.cz'
```

Comment:
Can you please change this to `localhost`? It is obviously meant to have no sense. By using localhost, we ensure that in case of error, tests won't make requests to real servers.

Reply:
Good point. Will check and change if possible.

Reply:
OK, fixed.